css-fonts/styles.css

```diff
@@ -0,0 +1,21 @@
+p {
+  font-family: 'Macondo',
+    cursive;
+  font-family: 'Roboto',
+    sans-serif;
+    line-height: 30px;
+}
+h1 {
+  font-family: 'Macondo',
+    cursive;
+  font-family: 'Roboto',
+    sans-serif;
+}
+
+h2 {
+    font-family: 'Macondo',
+    cursive;
+    font-family: 'Roboto',
+    sans-serif;
+
+}
```

Comment:
Make sure both Google Fonts are displayed somewhere on the page at the same time. Right now, `Macondo` is being overwritten by `Roboto` everywhere.